cypress/integration/project.spec.js

```diff
@@ -0,0 +1,490 @@
+      cy.get(TOP_BAR_SELECTOR).should('have.css', 'display', 'flex');
+    });
+
+    it('A classe header deve possuir a propriedade `background-color: rgb(50, 167, 145)`', () => {
+      cy.get(TOP_BAR_SELECTOR).should('have.css', 'background-color', 'rgb(50, 167, 145)');
+    });
+  });
+
+  describe('2 - Adicione o logotipo da Trybewarts com a classe `trybewarts-header-logo` no canto esquerdo da barra superior', () => {
+    it('Deve existir um elemento img com a classe `trybewarts-header-logo`', () => {
+      cy.get(TRYBEWARTS_LOGO_SELECTOR).should('exist');
+    });
+
+    it('O logotipo deve estar alinhado à esquerda dentro da barra verde', () => {
+      cy.get(TRYBEWARTS_LOGO_SELECTOR).should('be.leftAligned', TOP_BAR_SELECTOR);
+    });
+
+    it('O atributo src do logotipo deve apontar para images/trybewarts-header-logo.svg', () => {
+      cy.get(TRYBEWARTS_LOGO_SELECTOR).should('have.attr', 'src').should('equal', 'images/trybewarts-header-logo.svg');
+    });
+  });
+
+  describe('3 - Acrescente um formulário de autenticação no canto direito da barra superior contendo os inputs de login, de senha e um botão de entrar', () => {
+    it('Existe um formulário com a classe trybewarts-login', () => {
+      cy.get(TRYBEWARTS_LOGIN_FORM_SELECTOR).should('exist');
+    });
+
+    it('O formulário deve estar alinhado à direita dentro da barra verde', () => {
+      cy.get(TOP_BAR_SELECTOR).should('have.css', 'justify-content', 'space-between');
+    });
+
+    it('Existem dois inputs e um botão dentro do formulário', () => {
+      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} input`).should(($input) => {
+        expect($input).to.have.length(2);
+      });
+      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} button`).should('exist');
+    });
+
+    it('Os inputs deverão conter placeholders com as palavras "Login" e "Senha", respectivamente', () => {
+      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} input`).should(($input) => {
+        expect($input[0].placeholder).to.match(/Login/);
+        expect($input[1].placeholder).to.match(/Senha/);
+      });
+    });
+
+    it('O formulário deve ser um flex container', () => {
+      cy.get(TRYBEWARTS_LOGIN_FORM_SELECTOR).should('have.css', 'display', 'flex');
+    });
+
+    it('Ao clicar no botão com login ou senha inválidos, emite um alerta contendo o texto "Login ou senha inválidos."', () => {
+      const stub = cy.stub()
+      cy.on('window:alert', stub);
+      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} button`).click()
+        .then(() => {
+          expect(stub.getCall(0)).to.be.calledWith('Login ou senha inválidos.')
+        })
+    });
+
+    it('Ao clicar no botão com login ou senha válidos, emite um alerta contendo o texto "Olá, Tryber!"', () => {
+      const stub = cy.stub()
+      cy.on('window:alert', stub);
+      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} input:nth-child(1)`).type('tryber@betrybe.com');
```

Comment:
```suggestion
      cy.get(`${TRYBEWARTS_LOGIN_FORM_SELECTOR} input:nth-child(1)`).type('tryber@teste.com');
```

Melhor usar um email que com certeza nunca vai existir, just in case 😂